src/main/webapp/index.html

```diff
@@ -97,18 +97,17 @@ <h2>SANTIAGO MINGA</h2>
         <h1>UPLOAD IMAGE</h1>
         <div class="upload__container">
           <div class="upload__content">
-            <form action="/uploadImage" method="POST" enctype="multipart/form-data">
+            <form id="file-upload-form"action="/uploadImage" method="POST" enctype="multipart/form-data">
               <p>Post an animal picture!</p>
               <br/><br/>
               <label for="file-upload" class="submit__btn" >
                 Upload
               </label>
-              <input id="file-upload" type="file" name="image">
+              <input id="file-upload" type="file" name="image" onchange="submit()" >
               <!-- Add label and hide input type="file" for styling,
                 as the following solution gives and unwanted result:
                   <input type="button" name="image" class="submit__btn">  -->
               <br/><br/>
-              <button class="submit__btn" id="submit__btn">Show Image</button>
```

Comment:
Hey @SantiM1 is there still a bottom that appears on the screen. or does the form gets submitted automatically.

Reply:
It does get submited automatically and the red outline means it's a line I deleted, kinda confusing sometimes but just double checked either way lol

Reply:
Awesome, yep I know the red outline means the line is deleted, mention it because I saw that was the line for the bottom